### PR TITLE
Add abstract parameter.

### PR DESCRIPTION
**Pull request by gislenius**

### CODE REVIEW

**Comment by wcandillon** [comment]
@gislenius Can you document it?

File: queries/public/api/report-elements.jq [hunk 1]
```diff
@@ -47,7 +47,8 @@ declare  %rest:case-insensitive %rest:distinct  variable $label          as stri
 declare  %rest:case-insensitive                 variable $report         as string?  external;
 declare  %rest:case-insensitive %rest:distinct  variable $name           as string*  external;
 declare  %rest:case-insensitive                 variable $onlyNames      as boolean? external := false;
-declare  %rest:case-insensitive                 variable $onlyTextBlocks as boolean? external := false;
+declare  %rest:case-insensitive                 variable $onlyTextBlocks as boolean? external := ();
+declare  %rest:case-insensitive                 variable $abstract       as boolean? external := ();
```
Comment (wcandillon) [inline queries/public/api/report-elements.jq]: :+1:

File: swagger/queries.json [hunk 2]
```diff
@@ -2405,7 +2405,13 @@
...
             }, {
                 "name": "onlyTextBlocks",
-                "description": "Whether only text block report elements should be returned (default: false).",
+                "description": "Filters by text block/not text block (default: no filtering).",
+                "required": false,
+                "type": "boolean",
+                "paramType": "query"
+            }, {
+                "name": "abstract",
+                "description": "Filters by abstract/not abstract (default: no filtering).",
```
Comment (wcandillon) [inline swagger/queries.json]: :+1:

**Comment by wcandillon** [comment]
@gislenius You haven't tried the feature before marking it as ready for review.

**Comment by gislenius** [comment]
That's because I did it in my head and took my chance with a direct github edit ;-) Thanks for fixing the syntax error.

**Comment by wcandillon** [comment]
I must say that this edit in place feature in github is huge for productivity.

**Comment by gislenius** [comment]
@wcandillon feel free to merge ;-)

**Pull request closed without merging**